app/views/shared/check_answers/_no_link_items.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row" id="app-check-your-answers__<%= name %>">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m"><%= question %></h2>
+  </div>
+  <% if answers.present? %>
+    <div class="govuk-grid-column-one-third  govuk-summary-list--no-border" style="text-align: right;">
```

Comment:
Again - we shouldn't be applying styling directly to HTML unless there is no other way of doing it.